src/current/_includes/releases/v23.1/v23.1.14.md

```diff
@@ -0,0 +1,100 @@
+- Updated the **CPU Time** label to **SQL CPU Time** on the [DB Console](https://www.cockroachlabs.com/docs/v23.1/ui-overview-dashboard) and clarified the tooltip. [#116448][#116448]
+- Fixed an issue where all `AggHistogram`-powered metrics were not reporting quantiles properly in the [DB Console](https://www.cockroachlabs.com/docs/v23.1/ui-overview-dashboard). The list of affected metrics is:  
+    - `changefeed.message_size_hist` 
+    - `changefeed.parallel_io_queue_nanos` 
+    - `changefeed.sink_batch_hist_nanos` 
+    - `changefeed.flush_hist_nanos` 
+    - `changefeed.commit_latency` 
+    - `changefeed.admit_latency` 
+    - `jobs.row_level_ttl.span_total_duration` 
+    - `jobs.row_level_ttl.select_duration` 
+    - `jobs.row_level_ttl.delete_duration`  
+
+    Note that these histograms were only broken in the DB Console metrics features, but were not broken in the Prometheus-compatible endpoint, `/_status/vars`. [#114747][#114747]
+- In the **SQL Activity Transaction Details** page, users can now view a transaction fingerprint ID across multiple applications by specifying the app name in the `appNames` URL search parameter using a comma-separated encoded string of transaction fingerprint IDs. [#116102][#116102]
+
+<h3 id="v23-1-14-bug-fixes">Bug fixes</h3>
+
+- Fixed a bug where an active replication report update could get stuck in a retry loop on clusters with over 10000 ranges, which would prevent a node from shutting down cleanly. [#114242][#114242]
+- Fixed error handling for `GetFiles` so that it does not cause a nil pointer dereference. [#114829][#114829]
+- Fixed a bug that prevented the **SQL Activity** page from showing internal statements when the `sql.stats.response.show_internal.enabled` [cluster setting](https://www.cockroachlabs.com/docs/v23.1/cluster-settings) was set to `true`. [#114819][#114819]
+- Rolled back deletes no longer cause a discrepancy between computed stats and the actual stored values. [#113772][#113772]
+- Previously, CockroachDB could encounter an internal error when using the text format (as opposed to binary) when preparing statements with user-defined composite types. The bug was introduced in 23.1 and is now fixed. [#115063][#115063]
+- Fixed a bug that could cause cluster version finalization to contend with descriptor lease renewals on large clusters. Descriptor lease renewals previously had a higher priority than cluster upgrade finalization. Finalization now always has a higher priority than descriptor lease renewal. [#114672][#114672]
+- Fixed a bug that would cause a [prepared statement](https://www.cockroachlabs.com/docs/v23.1/sql-grammar#prepare_stmt) to fail if it references an enum as well as a table that has undergone a schema change. [#115174][#115174]
+- [Backups](https://www.cockroachlabs.com/docs/v23.1/backup) now evenly distribute their work across all replicas, including followers, regardless of leaseholder placement. [#115018][#115018]
```

Comment:
Does this apply to locality-aware backups too? cc/ @kathancox

Reply:
Checked with David T. -- he says the note is correct as is.